src/controllers/courseController.ts

```diff
@@ -101,6 +104,15 @@ export class CourseController implements FormController {
 			const modules: Module[] = response.locals.course.modules
 
 			for (let module of modules) {
+				if (module.type === Module.Type.FACE_TO_FACE) {
+					const events = _.get(module, 'events', [])
+
+					if (events) {
+            for (const event of (<FaceToFaceModule>module).events) {
```

Comment:
We only need to get the duration of the first event, not the duration of all events.

In terms of "first", ideally we should use the first event chronologically, but the first in the list would also be fine.